build.go

```diff
@@ -191,45 +201,6 @@ func (bf *BuildFactory) BuildConfigFromFlags(ctx context.Context, f *BuildFlags)
 	return b, nil
 }
 
-func Build(ctx context.Context, outWriter, errWriter io.Writer, appDir, builderImage, runImage, repoName string, publish, clearCache bool) error {
```

Comment:
I took the liberty of removing this as it seemed unused and duplicate of what the command does.

Reply:
Reverted and adapted as I just realized this is intended to be used as a library as well.